### DISCUSSION TITLE
Display "in house" frequency for a variant

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#3072] [Q&A] [answered]
Hello,

I have a local instance of Seqr. I would like to know if there is a possibility to display the "in house" frequency of a variant (if we already have the same variant in other samples that we have analyzed in Seqr).

Thank you in advance for your help.

**Comment by hanars** [comment, marked as answer]
If you joint call all your samples together, then the "Callset" frequency will reflect the frequency across all samples in seqr

Next year we plan to release an updated version of seqr with a different backend architecture, and that version of seqr will support showing the total frequency of a variant in seqr without joint calling

**Reply by bnouyou** [reply]
Thank you for your answer.
